src/components/Menu/MenuItem.tsx

```diff
@@ -7,7 +7,10 @@ import {eventBroker} from '../utils/event-broker';
 const b = block('menu');
 
 export interface MenuItemProps extends DOMProps, QAProps {
+    /** @deprecated use `iconStart` instead */
     icon?: React.ReactNode;
+    iconStart?: React.ReactNode;
+    iconEnd?: React.ReactNode;
```

Comment:
Please refactor README.md. We are need new props description and cases for landing and github parts. https://github.com/gravity-ui/uikit/blob/main/src/components/Menu/README.md#icon